Contents/Code/__init__.py

```diff
@@ -122,18 +122,34 @@ def getRecapItems(videos):
 		for video in videos:
 			bitrate = int(video["name"].split("_")[1][0:-1])
 			height = int(video["height"])
-			objects.insert(0, MediaObject(
-				container = Container.MP4,
-				video_codec = VideoCodec.H264,
-				audio_codec = AudioCodec.AAC,
-				video_resolution = height,
-				audio_channels = 2,
-				height = height,
-				width = int(video["width"]),
-				parts = [
-					PartObject(key=Callback(PlayRecap, url=video["url"]))
-				]
-			))
+			if int(Prefs['quality'][0:3]) == height:
+				objects.insert(0, MediaObject(
+					container = Container.MP4,
+					video_codec = VideoCodec.H264,
+					audio_codec = AudioCodec.AAC,
+					video_resolution = height,
+					audio_channels = 2,
+					height = height,
+					width = int(video["width"]),
+					parts = [
+						PartObject(key=Callback(PlayRecap, url=video["url"]))
+					]
+				))
```

Comment:
Isn't the point of the quality setting to avoid the menu with different qualities?
To do that, we should probably just return an array with a single array item here?

Reply:
Hmm, I don't see a menu with different qualities. I think I used to have that at some point, but now I don't see any way to choose the quality. Am I missing something?